### DISCUSSION TITLE
Link to url opens something else

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44014] [Mobile]
### Select Topic Area

Bug

### Body

1. Browse to the https://github.com/raku-community-modules organization
2. There's a url 🔗[**https://github.com/Raku/...ster/ADOPT-ME.md**](https://github.com/Raku/ecosystem/blob/master/ADOPT-ME.md)
3. Tap the URL expecting to load the linked thing

### Actual results

afaict, it opens https://github.com/Raku/ecosystem/ instead

**Comment by daniel-cues** [comment]
Not really, it opens https://github.com/Raku/ecosystem/blob/master/ADOPT-ME.md, I've just tried

**Reply by jsoref** [reply]
https://user-images.githubusercontent.com/2119212/214896658-ed6d11ec-1e47-4590-8e1e-8ab8aa7969a4.mp4
